cmd/build.go

```diff
@@ -66,6 +74,13 @@ func beforeBuild(ctx *cli.Context) error {
 		}
 	}
 
+	if !ctx.IsSet("remote-save") {
+		if len(ctx.StringSlice("remote-save-tag")) != 0 {
+			// Cannot set upload tags if uploading to storage URL is disabled
+			return fmt.Errorf("--remote-save-tag requires --remote-save")
+		}
+	}
+
```

Comment:
Seems like we also need a check for when you supply `--remote-save` but forget `--remote-save-tag`; or better yet, can we just have `--remote-save-tag` and drop `--remote-save` all together? You'd have to supply the full URL, but then you could save to multiple repositories, which might be nice.

Reply:
There's 4 cases right now:
1. File is in Git and --remote-save-tag is sent -> both commit-id tag and the tag from command line are used 
2. File is in Git and  --remote-save-tag is not sent -> only the commit-id tag is used
3. File is not in Git and --remote-save-tag is sent -> only the tags from command line are used
4. File is not in Git and the --remote-save-tag is not sent is not set -> the image is not saved

Because of #2, if we drop the --remote-save it means we always try to save with commit-id, and that would mean there's no control on how to disable the feature, unless the user edits all the relevant stacker.yaml files locally.

Please explain the part about having to supply the full URL. Are you suggesting we should send it from command line, and only if we send it from command line we save? 
In this case how do we distinguish between images we want to save and the others?

Reply:
Sorry, you're right. I think we can just get rid of `--remote-save` entirely, then, and if you've supplied a URL then you get the caching, and if you've supplied `--remote-save-tag`, you get additional tags. Does that sound reasonable?

Reply:
Done